hash/apple2_flop_misc.xml

```diff
@@ -955,6 +955,24 @@ Generated by SLIST 0.2.1
 		</part>
 	</software>
 
+	<software name="antim18">
+		<description>Anti-M (version 1.8)</description>
+		<year>2020</year>
+		<publisher>4AM</publisher>
+		<info name="release" value="2020-08-11"/>
+		<!--
+		* support "Sensible Speller" and "Typing Tutor"
+		* add some compatibility fixes for Virtual ][
+		-->
+
+		<part name="flop1" interface="floppy_5_25">
+			<dataarea name="flop" size="143360">
+				<rom name="anti-m-v1.8-2020-11-10.dsk" size="143360" crc="2089c8ae" sha1="896d37ed95339b0ac247e5b40bbd5dfdfd40ed95" />
+			</dataarea>
+		</part>
+	</software>
```

Comment:
These should really have parent/clone relationships.

Reply:
That's a valid point I hadn't considered. I've just done that and I'm putting through a commit for that right now.